content/en/docs/concepts/architecture/nodes.md

```diff
@@ -111,9 +111,9 @@ on a Node.
 
 ### Info
 
-General information about the node, such as kernel version, Kubernetes version
-(kubelet and kube-proxy version), Docker version (if used), OS name.
-The information is gathered by Kubelet from the node.
+Describes the general information about the node, such as kernel version, Kubernetes version
```

Comment:
Replace with:
> Describes general information about the node, such as kernel version and Kubernetes version.

Reply:
I think `full stop` is not required after `Kubernetes version` as it is the same line.